pkg/alertmanager/amcfg.go

```diff
@@ -2388,20 +2446,21 @@ func (tc *webexConfig) sanitize(amVersion semver.Version, logger *slog.Logger) e
 	return tc.HTTPConfig.sanitize(amVersion, logger)
 }
 
-func (jc *jiraConfig) sanitize(amVersion semver.Version, logger *slog.Logger) error {
-	jiraConfigAllowed := amVersion.GTE(semver.MustParse("0.28.0"))
-	if !jiraConfigAllowed {
-		return fmt.Errorf(`invalid syntax in receivers config; jira integration is available in Alertmanager >= 0.28.0`)
+func (tc *jiraConfig) sanitize(amVersion semver.Version, logger *slog.Logger) error {
+	JiraAllowed := amVersion.GTE(semver.MustParse("0.28.0"))
+	if !JiraAllowed {
+		return fmt.Errorf(`invalid syntax in receivers config; Jira integration is available in Alertmanager >= 0.28.0`)
 	}
 
-	if jc.Project == "" {
-		return fmt.Errorf("missing project in jira_config")
+	if tc.Project == "" {
+		return fmt.Errorf("mandatory field %q is empty", "project")
 	}
-	if jc.IssueType == "" {
```

Comment:
Seems you're removing the validation for IssueType, I would think is required as per https://github.com/prometheus/alertmanager/blob/main/config/notifiers.go#L926

Reply:
Thanks @heliapb
Fixed
Add IssueType as mandatory field